web/modules/custom/joinup_subscription/config/install/field.field.user.user.field_user_frequency.yml

```diff
@@ -14,7 +14,9 @@ label: 'Notification frequency'
 description: 'How often do you prefer to receive status updates on new and/or updated content on Joinup?'
 required: true
 translatable: false
-default_value: {  }
+default_value:
+  -
+    value: daily
```

Comment:
This is setting the "daily digest" as the default option. This is just a best guess, as far as I know this isn't yet described as a functional requirement.